docs(frontend): adding small intros (#1029)
zama-ai/concrete: docs/explanations/security.md

# Security

> This document describes some security concepts around FHE, such that how we can generate parameters which are both secure and correct.

Reviewer: ```suggestion
This document describes some security concepts around FHE that can help you generate parameters that are both secure and correct.
```
